src/components/Comments.vue

```diff
@@ -222,7 +380,19 @@ export default class Comments extends Vue {
   }
 
   get rootComments() {
-    return this.comments.filter(comment => comment.parent == null);
+    return this.comments.filter(
+      comment => comment.parent == null && comment.isHidden != true
```

Comment:
`!= true` is redundant. Should be `!comment.isHidden`

Reply:
fixed